.github/ISSUE_TEMPLATE.md

```diff
@@ -1,8 +1,25 @@
 <!--
-File an issue if you think you've found a bug. Please describe
 
-1. How can it be reproduced
-2. What was expected
-3. What actually occurred
-4. What version of the involved component
--->
\ No newline at end of file
+Template: BUG REPORT
+
+This is an issue for a bug report.
+You should label `bug-report`.
+
+
+If your purpose is different, please visit following links.

```

Comment:
Suggestion: "If you issue is not a bug report, please visit one of the following links."

Reply:
Did you mean "If your issue is..."? 🙄

Reply:
yes of course - I think I was getting tired at that point 👍

Reply:
👍 👍 👍